docs/source/license.rst

```diff
@@ -0,0 +1,10 @@
+Licence
```

Comment:
Usually we have a LICENSE in the main repo directory. There also should be a auto-generated one. Can we amend that one instead and re-use here?

Reply:
Ah right, this would make much more sense. Will change this in the next commit!